src/client/app/components/about.ts

```diff
@@ -29,17 +37,62 @@ import { environment, storageSize, Strings, storage } from '../helpers';
     `
 })
 
-export class About {
+export class About implements AfterViewInit {
     @Input() show: boolean;
     @Output() showChange = new EventEmitter<boolean>();
+
+    cache = [
+        Strings.aboutStorage,
+        storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets),
+        storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)
+    ].join('\n');
+
     config = {
         build: environment.current.build,
         editorUrl: environment.current.config.editorUrl,
     };
 
-    cache = `
-    ${Strings.aboutStorage}
-    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets)}
-    ${storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)}
-    `;
+    configs = [];
+
+    selectedConfig = '';
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) { }
+
+    ngAfterViewInit() {
+        this.configs.push(
+            { name: 'production', value: 'Production' },
+            { name: 'insiders', value: 'Beta' },
+            { name: 'edge', value: 'Alpha' },
+        );
+        if (environment.current.config.name === config['local'].name) {
```

Comment:
Now that we are passing originEnvironment it should be easy to add localhost to the list if user navigated from localhost to a different environment.